lib/alice/adapters.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Alice.Adapters do
+  @moduledoc false
+
+  def start_link(opts \\ [start_adapter: true]) do
+    if opts[:start_adapter], do: selected_adapter.start_link
+    {:ok, selected_adapter}
+  end
+
+  @doc """
+  Selects adapter, defaults to Slack
+  """
+  def selected_adapter do
+    Application.get_env(:alice, :adapter, Alice.Adapters.Slack)
```

Comment:
This is another thing we'll be attacking in the next version of Alice, the ability to start multiple instances of a bot with different configuration settings or adapters.